webargs/core.py

```diff
@@ -270,7 +270,10 @@ def parse_arg(self, name, field, req, locations=None):
         """
         location = field.metadata.get("location")
         if location:
-            locations_to_check = self._validated_locations([location])
+            if isinstance(location, tuple) or isinstance(location, list):
```

Comment:
This can be simplified to

```suggestion
            if isinstance(location, (list, tuple)):
```